### PR TITLE
feat: add json schema generators to every contract

### PR DESCRIPTION
**Pull request by cgorenflo**

### CODE REVIEW

File: contracts/voting-verifier/src/msg.rs [hunk 1]
```diff
@@ -61,7 +61,7 @@ pub enum QueryMsg {
     #[returns(Poll)]
     GetPoll { poll_id: PollID },
 
-    #[returns(Vec<(CrossChainId, bool)>)]
+    #[returns(Vec<(connection_router::state::CrossChainId, bool)>)]
```
Comment (fish-sammy) [inline contracts/voting-verifier/src/msg.rs:64]: What's the reason for this change?

Comment (cgorenflo) [reply contracts/voting-verifier/src/msg.rs:64]: trying to build the json schema showed that the unqualified name couldn't be resolved

**Review by cjcobb23 — APPROVED** [review]
Looks good
